docs/environment-variables.md

```diff
@@ -63,25 +63,38 @@ The OAuth token secret, generated by Launchpad.
 ## GitHub settings
 
 ### GITHUB_API_ENDPOINT
-- **Example**: `http://localhost:4000/github`, `https://api.github.com`
+- **Example**: `http://localhost:4000/github`
+- **Example**: `https://api.github.com`
 
 The base URL for the GitHub API.
 
-### GITHUB_USERNAME
-- **Example:** katiefenn
-- **DEPRECATED**
```

Comment:
`GITHUB_USERNAME` and `GITHUB_PASSWORD` finally deprecated in favour of Oauth tokens 💃